pkg/codegen/nodejs/gen.go

```diff
@@ -726,7 +726,6 @@ func (mod *modContext) genResource(w io.Writer, r *schema.Resource) error {
 				}
 				if arg.IsRequired() {
 					argsOptional = false
-					break
```

Comment:
👿 

I've spent so many years in FP I literally fear these keywords `break` and `continue`. It looks like the fear is not entirely unjustified.